### PR TITLE
Fixed broken links to support and contributing pages in READMEs

### PR DESCRIPTION
**Pull request by djcoleman**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Comment by djcoleman** [comment]
The camel-example-hello-grpc-server test failed to build, but I don't think the doc changes had anything to do with that. Please can you merge this PR if you're happy with it as I don't have write access to the repository.

**Comment by oscerd** [comment]
Thanks for this.

**Pull request merged**
